src/main/java/com/google/devtools/build/lib/packages/SkylarkSemanticsOptions.java

```diff
@@ -62,6 +62,16 @@
 
   // <== Add new options here in alphabetic order ==>
 
+  // TODO(gregce): remove license checking completely from Bazel. aiuto@ is working on replacing
+  // this with a new and more useful model.
+  @Option(
+      name = "check_third_party_targets_have_licenses",
+      defaultValue = "true",
```

Comment:
I believe setting this flag to true by default makes it an incompatible change, or am I missing something?

Reply:
Without this flag (the status quo), this check already happens. This checks that every rule in `third_party` declares a `licenses()`. Most Bazel users don't find this helpful.

Confusingly, we also have `--check_licenses`, which is different, and is not on by default. That's what actually checks that the rules you're building are compatible with those licenses.

Reply:
Ah, gotcha